scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -1004,7 +1004,19 @@ s32 BPF_STRUCT_OPS(rusty_select_cpu, struct task_struct *p, s32 prev_cpu,
 		struct dom_ctx *domc;
 		struct bpf_cpumask *tmp_direct_greedy, *node_mask;
 
-		if (!(domc = lookup_dom_ctx(dom_id)))
+		/*
+		* CPU may be offline e.g. CPU was removed via hotplugging and scheduler
```

Comment:
nit: most of the other commends keep alignment of `*` chars

Reply:
ugh, can't believe i missed this